ts/project.test.ts

```diff
@@ -0,0 +1,52 @@
+import { Check } from "./check.ts";
+import { Executable } from "./executable.ts";
+import { Project } from "./project.ts";
+
+const assertTypeIsCheck = (_c: Check) => {};
```

Comment:
I wanted to make it a little bit more self-explanatory that all these definitions here are just for asserting on type checks and not being executed. I added 138ec9e9 for that. TIOLI!